Sources/AccioKit/Services/XcodeProjectIntegrationService.swift

```diff
@@ -244,17 +244,25 @@ final class XcodeProjectIntegrationService {
     }
 
     private func verifyBundleVersion(of product: FrameworkProduct) throws {
```

Comment:
Why is this called "verify" when it also actually updates the Info.plist file? Maybe it should be `ensureBundleVersionExistence`.

Reply:
Ask already hinted in the commit message I agree and will change it.